src/components/WeatherCardComponent/WeatherCardComponent.jsx

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Were you disabling a rule? This comment also needs to be removed

Reply:
already fixed